### PR TITLE
Update technical documentations to help onboarding new developers

### PR DESCRIPTION
**Pull request by BtcContributor**
Technical docs updated to help the onboarding of new developers as I had some difficulties when I recently started joining the project.

Hope this could help Bisq in a meaningful way :)

### CODE REVIEW

File: docs/testing.md [hunk 1]
```diff
@@ -13,6 +13,7 @@ In order to take part in the testing process, you will need to do the following:
 ## Communication Channels
 
 If you would like to discuss and/or contribute to Bisq's testing effort, join us in the #testing channel within the [Bisq Keybase team](https://keybase.io/team/bisq).
+Here you could also request access to TestPad.
```
Comment (ripcurlx) [inline docs/testing.md]: Maybe link to `bisq.ontestpad.com`

**Comment by ripcurlx** [comment]
@BtcContributor Could you please resolve the conflict and remove the blank line Codacy is complaining about. Thanks!

**Comment by BtcContributor** [comment]
@ripcurlx I am not sure how to resolve conflict. I see no conflict on git from command line and if I try to solve it from Github the merge commit will not be signed. What can I do? Do you have any suggestions? Thanks!

**Review by ripcurlx — APPROVED** [review]
utACK

**Pull request merged**